<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurveyPropertiesBase</class>
 <widget class="QDialog" name="SurveyPropertiesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Survey properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mSurveyLabel">
     <property name="text">
      <string>Survey</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="mSurveyLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mProjectCodeLabel">
     <property name="text">
      <string>Project code</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mProjectCodeLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mDateSLabel">
     <property name="text">
      <string>date_s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="mDateSEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mDateFLabel">
     <property name="text">
      <string>date_f</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="mDateFEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mContactNameLabel">
     <property name="text">
      <string>Contact name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="mContactNameLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="mAreasLabel">
     <property name="text">
      <string>Areas</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="mAreasLineEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="mMainSppLabel">
     <property name="text">
      <string>mainspp</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="mMainSppLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="mCommentsLabel">
     <property name="text">
      <string>Comments</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="mCommentsLIneEdit"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SurveyPropertiesBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SurveyPropertiesBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
